src/main/java/com/scalyr/integrations/kafka/AddEventsClient.java

```diff
@@ -182,15 +190,17 @@ public AddEventsClient(String scalyrUrl, String apiKey, long addEventsTimeoutMs,
   /**
    * Call Scalyr addEvents API with {@link #addEventsTimeoutMs} using exponential backoff.
    * @param addEventsPayload byte[] addEvents payload
+   * @param rawPayloadSize long raw addEvents payload size before the compression.
```

Comment:
Perhaps ``uncompressedPayloadSize`` would be a better and more explicit argument name?

Reply:
yes, agree